guide/designing-products/common-user-flows.md

```diff
@@ -251,13 +251,19 @@ See the [onboarding section](/guide/onboarding/introduction/) to learn more on t
+
 Not all wallets are the same as new standards have emerged over time. So some software may create wallets via outdated technical formats, and others allow users to choose different technical formats during wallet creation for wider compatibility with other software. [Wallets Recovery](https://github.com/nvk/wallets-recovery){:target="_blank"} provides a great overview of different implementations and how nuanced some of the differences are. Generally, these differences are very difficult to understand for regular users and should either be automatically handled by the system, or explained in layman terms.
 
-One of the most complex users flows in Bitcoin is the creation of multisignature wallets. Multisignature means that there are multiple keys that control the wallet. All keys are required for the creation of the wallet, and users can choose the number of keys required to sign transactions. This is typically done to increase security because it avoids having a single point of failure. For a hacker to get access to 2 out of total of 3 keys that are geographically distributed is much harder than if there is just a single key. While a single user can more easily set up a multisignature wallet with one computer and multiple hardware wallets, it is harder when multiple users hold keys. In that scenario, a file needs to be sent around for each participant to sign. Some wallet software providers hosts servers that seamlessly connect these users. However, this provides another potential attack vector and introduces risk.
+Most modern wallet applications should support the Lightning network in addition to the base layer. While there are different options for how the applications interact with a Lightning network node, an HD wallet works fine for storing the required keys.
+
+Creating multi-key wallets (or multi-signature / multi-sig), can be one of the most complex user flows in Bitcoin. This setup means that several keys control the wallet. All keys are required for the creation of the wallet, and users can choose the number of keys required to sign transactions. This is typically done to increase security. For a hacker to get access to several keys that are geographically distributed is much harder than if there is just a single key. While a single user can somewhat easily set up a multisignature wallet with a computer or mobile device and multiple wallets in their possession, it is harder when different people hold keys. In that scenario, a file needs to be sent around for each participant to sign. Some wallet software providers host servers that seamlessly connect these users. However, this provides another potential attack vector and introduces risk.
```

Comment:
Is multi-sig a 'common user flow'? I'd argue not. Lot of content here for something that isn't our primary focus imo.

Reply:
Agree, I would be up for cutting most or all of this.
As this PR was focused on changes for Lightning I made mostly readability changes.

Reply:
This was written before a lot of the other content existed. We can toss it (or merge useful parts into other pages), and instead link out to the [Multi-key](https://bitcoin.design/guide/private-key-management/multi-key/), [Creating a new wallet](https://bitcoin.design/guide/onboarding/creating-a-new-wallet/) page as well as the [case studies](https://bitcoin.design/guide/case-studies/) ([Savings account](https://bitcoin.design/guide/case-studies/savings-account/) covers multisig setup well, for example). Having all this other content can really help us turn this page into more of an overview and then just link out.

Reply:
Agreed, I think perhaps just mentioning that multi-sig is a thing and linking to other pages that have more detailed multi-sig information would be sufficient.

Reply:
Replaced with simple mention and link to the multi-key page.